yarn.lock

```diff
@@ -4178,11 +4178,6 @@
   dependencies:
     "@types/node" "*"
 
-"@types/caseless@*":
```

Comment:
Not sure what's going on here, looks like the yarn lockfile could be slightly outdated?